src/driver.ts

```diff
@@ -3,4 +3,5 @@ import { BoundStatement, Params, ResultSet } from "./shared-types.js";
 export interface Driver {
     execute(stmt: string, params?: Params): Promise<ResultSet>;
     transaction(stmts: (string | BoundStatement)[]): Promise<ResultSet[]>;
+    // TODO: add a `close()` method
```

Comment:
The `Driver` interface does not have a `close()` method. It should have one, so that the driver can clean up (i.e., close the WebSocket or the SQLite connection).